src/IPNFT.sol

```diff
@@ -46,6 +46,7 @@ contract IPNFT is ERC721URIStorageUpgradeable, ERC721BurnableUpgradeable, IReser
 
     event Reserved(address indexed reserver, uint256 indexed reservationId);
     event IPNFTMinted(address indexed owner, uint256 indexed tokenId, string tokenURI, string symbol);
+    event IPNFTPOI(uint256 indexed tokenId, bytes poi);
```

Comment:
don't throw an extra event. for the IPNFT it should not matter if it's based on a poi